Library/OcAppleKernelLib/CommonPatches.c

```diff
@@ -884,6 +884,55 @@ PatchForceInternalDiskIcons (
   return Status;
 }
 
+STATIC
+UINT8
+mIONVMeFamilyPatchFind[] = {
+  0x45, 0x78, 0x74, 0x65, 0x72, 0x6E, 0x61, 0x6C
+};
+
+STATIC
+UINT8
+mIONVMeFamilyPatchReplace[] = {
+  0x49, 0x6E, 0x74, 0x65, 0x72, 0x6E, 0x61, 0x6C
+};
+
+STATIC
+PATCHER_GENERIC_PATCH
+mIONVMeFamilyPatch = {
+  .Comment      = DEBUG_POINTER ("IONVMeFamily"),
+  .Base         = NULL,
+  .Find         = mIONVMeFamilyPatchFind,
+  .Mask         = NULL,
+  .Replace      = mIONVMeFamilyPatchReplace,
+  .ReplaceMask  = NULL,
+  .Size         = sizeof (mIONVMeFamilyPatchFind),
+  .Count        = 1,
+  .Skip         = 0
+};
+
+STATIC
+EFI_STATUS
+PatchForceInternalDiskIconsNVME (
+  IN OUT PATCHER_CONTEXT    *Patcher,
+  IN     UINT32             KernelVersion
+  )
+{
+  EFI_STATUS          Status;
+
+  if (Patcher == NULL) {
```

Comment:
You must add version checking here similar to PatchUsbXhciPortLimit3. This patch should not return failure on systems where NVMe support is not present. This will break our tests for all the kernel versions.

Reply:
Done. Please check if it is ok.